cmd/environment.go

```diff
@@ -77,7 +77,15 @@ var environmentCreateCmd = &cobra.Command{
 		validateConsortiumID("environment")
 
 		client := getNewClient()
-		environment := kld.NewEnvironment(name, desc, provider, consensus, multiRegion, blockPeriod)
+		accountBalances := map[string]string{}
```

Comment:
can I suggest using a single param to accomplish this:
`--pre-fund ["0xf601c8a58a738c1055094d0cf3018266d562c4a5:1000","0xb68965e17da0A5602A7721053accdB83067726f0:2000000"]`

Reply:
sounds good, can go in as-is then.